docs/iris/src/whitepapers/missing_data_handling.rst

```diff
@@ -0,0 +1,81 @@
+=============================
+Missing Data Handling in Iris
+=============================
+
+This document provides a brief overview of how Iris handles missing data values
+when datasets are loaded as cubes, and when cubes are saved or modified.
+
+A missing data value, or fill-value, defines the value used within a dataset to
+indicate that data point is missing or not set.
+This value is included as part of a dataset's metadata.
+
+For example, in a gridded global ocean dataset, no data values will be recorded
+over land, so land point will be missing data.
+In such a case, land points could be indicated by being set to the dataset's
+missing data value.
+
+
+Loading
+-------
+
+On load, any fill-value or missing data value defined in the loaded dataset
+should be used as the ``fill_value`` of the NumPy masked array data attribute of the
+:class:`~iris.cube.Cube`. This will only appear when the cube's data is realised.
+
+
+Saving
+------
+
+On save, the fill-value of a cube's masked data array is **not** used in saving data.
+Instead, Iris always uses the default fill-value for the fileformat, *except*
+when a fill-value is specified by the user via a fileformat-specific saver.
+
+For example::
+
+    >>> iris.save(my_cube, 'my_file.nc', fill_value=-99999)
+
+.. note::
+    Not all savers accept the ``fill_value`` keyword argument.
+
+Iris will check for and issue warnings of fill-value 'collisions'.
+This basically means that whenever there are unmasked values that would read back
+as masked, we issue a warning and suggest a workaround.
```

Comment:
awesome

Reply:
@cpelley that's my kind of review comment 😀